torch/csrc/jit/passes/quantization.cpp

```diff
@@ -136,20 +136,16 @@ bool isFunctionNode(Node* n,
 // the quantization parameters for output given inputs
 std::vector<size_t> getGeneralOpTensorInputIndexes(Node* n) {
   std::vector<std::string> single_input_aten_funcs = {
-    "adaptive_avg_pool2d",
-    "max_pool2d",
-    "avg_pool2d",
-    "flatten",
-    "max",
-    "min",
-    "mean",
-    // TODO: sort returns a tuple of Tensors, we have
-    // to extend the API to support that
-    // "sort",
-    "__interpolate",
```

Comment:
could you add the corresponding ops to this list? I think it might be better to just put this change in the same PR

Reply:
I don't know the difference between "single_input_call_funcs" and "single_input_aten_funcs"

Reply:
Where should I be putting the ops?